includes/Stories_Renderer/Carousel_Renderer.php

```diff
@@ -57,10 +57,11 @@ public function assets() {
 		parent::assets();
 
 		if ( ! $this->is_amp_request() ) {
-			// Enqueue amp runtime script and amp-carousel script to show amp-carousel on non AMP pages.
-			wp_register_script( 'amp-runtime-script', 'https://cdn.ampproject.org/v0.js', [], 'v0', true );
-			wp_register_script( 'amp-carousel-script', 'https://cdn.ampproject.org/v0/amp-carousel-0.2.js', [ 'amp-runtime-script' ], 'v0', true );
-			wp_enqueue_script( 'amp-carousel-script' );
+
+			wp_enqueue_script( 'glider-js', WEBSTORIES_PLUGIN_DIR_URL . 'includes/assets/lib/glider-js/glider.min.js', [], '1.7.4', true );
+			wp_enqueue_style( 'glider-css', WEBSTORIES_PLUGIN_DIR_URL . 'includes/assets/lib/glider-js/glider.min.css', [], '1.7.4' );
```

Comment:
Let's not copy the whole `glider-js` library to `includes/assets/lib`

Instead, we can install it from [npm](https://www.npmjs.com/package/glider-js)

```
npm install --save glider-js
```

Then, we can add a build step in the `webStoriesScripts` config in `webpak.config.cjs` using [`webpack-copy-plugin`](https://webpack.js.org/plugins/copy-webpack-plugin/):

```js
new CopyWebpackPlugin([
  { from: 'node_modules/glider-js/path/to/script/**', to: 'includes/assets/lib/glider-js' },
],
```

Reply:
Along with the js/CSS, we will also need to copy the LICENSE there is right?

We'll need three files from the `node_module`
- glider.js
- glider.css
- LICENSE.txt

Reply:
again, @swissspidy if we are adding the package with npm and import it in the required JS, I don't think we will need to copy the files. The build process will take care of outputting the files in the required bundles.

Reply:
Oh, right! Even better if we don't need to copy the files.